### PR TITLE
Use dc:creator instead of author for RSS 2.0 outputs

### PR DESCRIPTION
**Pull request by Alkarex**
https://github.com/FreshRSS/FreshRSS/issues/2538#issuecomment-536227043

### CODE REVIEW

File: app/views/index/rss.phtml [hunk 1]
```diff
@@ -17,7 +17,7 @@ foreach ($this->entries as $item) {
 				$authors = $item->authors();
 				if (is_array($authors)) {
 					foreach ($authors as $author) {
-						echo "\t\t\t" , '<author>', $author, '</author>', "\n";
+						echo "\t\t\t" , '<dc:creator>', $author, '</dc:creator>', "\n";
```
Comment (Frenzie) [inline app/views/index/rss.phtml:20]: Is `$author` here supposed to be a (pen) name or was this merely a trickle-down effect?

**Comment by Alkarex** [comment]
In RSS 2.0, `<author>` expects an e-mail address https://cyber.harvard.edu/rss/rss.html#ltauthorgtSubelementOfLtitemgt
In most feeds I see, this is not the case, but rather a name

**Comment by Frenzie** [comment]
Right, that's what I mean by trickle-down effect. So it comes from the original feed then.

**Comment by Alkarex** [comment]
Yes

**Pull request merged**